test/kubemark/start-kubemark.sh

```diff
@@ -261,16 +250,21 @@ eventer_mem_per_node=500
 eventer_mem=$((200 * 1024 + ${eventer_mem_per_node}*${NUM_NODES:-10}))
 sed -i'' -e "s/{{EVENTER_MEM}}/${eventer_mem}/g" "${RESOURCE_DIRECTORY}/addons/heapster.json"
 
+# Create kubemark namespace.
 "${KUBECTL}" create -f "${RESOURCE_DIRECTORY}/kubemark-ns.json"
-"${KUBECTL}" create -f "${KUBECONFIG_SECRET}" --namespace="kubemark"
-"${KUBECTL}" create -f "${NODE_CONFIGMAP}" --namespace="kubemark"
+# Create configmap for configuring hollow- kubelet, proxy and npd.
+"${KUBECTL}" create configmap "node-configmap" --namespace="kubemark" \
+  --from-literal=content.type="${TEST_CLUSTER_API_CONTENT_TYPE}" \
```

Comment:
It seems it was copied from above, but I'm not 100% sure it is correct. What is this "content.type" used for?